pylzy/tests/api/v1/mocks.py

```diff
@@ -147,7 +150,7 @@ def ReadStdSlots(
 
 
 class StorageRegistryMock(StorageRegistry):
```

Comment:
`register_default_storage` is not a base interface method? Is it really needed?

Reply:
It is a base interface method, I forget to add it in mock.

Reply:
fixed